scope/fritz.py

```diff
@@ -71,6 +71,7 @@ def get_stats(G, ids):
 MAX_ATTEMPTS = config['fritz']['max_attempts']
 SLEEP_TIME = config['fritz']['sleep_time']
 fritz_token = config['fritz']['token']
+default_catalog = config['kowalski']['collections']['sources']
```

Comment:
Should this be a get in case it is not defined?

Reply:
Good idea, I'll make that change.